public/components/content-list-item/templates/preview.html

```diff
@@ -0,0 +1,8 @@
+<td class="content-list-item__field--preview">
```

Comment:
I'd keep the `td` co-located with it's parent `table` to make this component more reusable.

Reply:
On second look, it looks like this is a pattern we've already established. Ignore this comment then.